client/my-sites/plugins/plugin-details-sidebar/index.jsx

```diff
@@ -91,7 +95,11 @@ const PluginDetailsSidebar = ( {
 				id="support"
 				icon={ { src: support } }
 				title={ translate( 'Support' ) }
-				description={ translate( 'Live chat support 24x7.' ) }
+				description={
+					isAnnualPeriod
+						? translate( 'Live chat support 24x7.' )
+						: translate( 'Unlimited Email Support' )
```

Comment:
Nitpicking: I'm not sure about this `.` in the end of `'Live chat support 24x7.'`. 
It is not present on the other place and none of the `'Unlimited Email Support'` has this trailing dot.

I would go for removing it.

Reply:
Good catch. That message was like that before my change, but I agree it is not consistent with other appearances of the same message, so I will go ahead and remove the `.`